docs/gettingstarted.rst

```diff
@@ -1,4 +1,50 @@
 Getting started
 ===============
 
-FIXME
+To be able to deploy the REANA system, the Kubernetes configuration must be generated. In order to do so, we should do the following.
+
+.. code-block:: sh
+
+   $ python setup.py install
+   $ reana-resources-k8s build-manifests
```

Comment:
BTW shall we use "build-manifests" terminology? Wondering because we usually use "create-instance" etc...